lib/galaxy/tools/deps/resolvers/conda.py

```diff
@@ -40,6 +40,19 @@
 DEFAULT_BASE_PATH_DIRECTORY = "_conda"
 DEFAULT_CONDARC_OVERRIDE = "_condarc"
 DEFAULT_ENSURE_CHANNELS = "iuc,bioconda,r,defaults,conda-forge"
+CONDA_SOURCE_CMD = """[ "$CONDA_DEFAULT_ENV" = "%s" ] ||
+MAX_TRIES=3
+COUNT=0
+while [ $COUNT -lt $MAX_TRIES ]; do
+    . %s '%s' > conda_activate.log 2>&1
+    if [ $? -eq 0 ];then
+       break
+    else
+        sleep 10s
+        let COUNT=COUNT+1
+    fi
+done """
```

Comment:
That's right, it will just continue and fail with command not found, which is OK, no ?